core/storage/hamt/hamt.hpp

```diff
@@ -160,6 +160,9 @@ namespace fc::storage::hamt {
      */
     outcome::result<CID> flush();
 
+    /// Get root CID if flushed, throw otherwise
+    const CID &cid() const;
```

Comment:
Why do you need it?

Reply:
To get root cid separately from flush, as OUTCOME_TRY requires new name for flushed cid